databricks/koalas/indexes.py

```diff
@@ -1484,6 +1484,16 @@ def delete(self, loc):
                     "index {} is out of bounds for axis 0 with size {}".format(loc, length)
                 )
             loc = [loc]
+        # pandas with numpy>=1.19.0 raises Exception if the item in the list exceeds the length.
+        # Because the behavior of `np.delete` has been changed from numpy 1.19.0
+        # which pandas uses directly.
+        elif LooseVersion(np.__version__) >= LooseVersion("1.19.0"):
```

Comment:
I'm wondering whether we should switch the behavior based on the underlying numpy?
I guess we can just raise the `IndexError` when out of bounds?

Reply:
+1